mod-source-record-manager-server/src/main/java/org/folio/rest/impl/ChangeManagerImpl.java

```diff
@@ -51,6 +52,25 @@ public ChangeManagerImpl(Vertx vertx, String tenantId) { //NOSONAR
     this.tenantId = TenantTool.calculateTenantId(tenantId);
   }
 
+
+  @Override
+  public void deleteChangeManagerJobExecutions(DeleteJobExecutionsDto entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    vertxContext.runOnContext(c -> {
+      try {
+        jobExecutionService.deleteJobExecutionsByIds(entity.getIds(), tenantId)
+          .map(optionalJobExecution -> !optionalJobExecution.booleanValue()?
+            new BadRequestException(format("JobExecution with id '%s' was not found", entity.getIds())): optionalJobExecution)
+          .map(DeleteChangeManagerJobExecutionsResponse::respond200WithApplicationJson)
+          .map(Response.class::cast)
+          .otherwise(ExceptionHelper::mapExceptionToResponse)
+          .onComplete(asyncResultHandler);
+      } catch (Exception e) {
+        LOGGER.error("Failed to delete JobExecutions by ids {} : Exception : ", entity.getIds(), e);
+        asyncResultHandler.handle(Future.succeededFuture(ExceptionHelper.mapExceptionToResponse(e)));
```

Comment:
nit: should this be a Future.failedFuture instead?

Reply:
@SatishkumarPurswani what do you think of this?

Reply:
Hi @okolawole-ebsco - I think, we are returning the `Future.succeededFuture` as we want to pass the status code and the message along with it, whereas `Future.failedFuture` takes either String or Throwable which will not suffice the need.

Reply:
Great point! Yeah, it is good as it is.